pyomo/contrib/parmest/tests/test_parmest.py

```diff
@@ -32,8 +32,12 @@
 ipopt_available = SolverFactory("ipopt").available()
 pynumero_ASL_available = AmplInterface.available()
 testdir = this_file_dir()
+SET_GLOBAL_SEED = True
 
-
+if SET_GLOBAL_SEED:
```

Comment:
Alternate idea: either here or in the examples file (or somewhere else the Pyomo team recommends), define

`_RANDOM_SEED_FOR_TESTING = 524`

Then in all the files that use this random seed, you can import this value.

Then in this file:
```
if _RANDOM_SEED_FOR_TESTING:
   np.random.seed(_RANDOM_SEED_FOR_TESTING)

This will mean someone only sets the testing random seed in one spot.